solana/program_v2/src/AdminService.ts

```diff
@@ -109,14 +108,16 @@ export class AdminService extends AbstractService {
       authority,
     });
   }
-
   createNetwork(
     data: CreateNetworkData = {
       authThreshold: 1,
       passExpireTime: 16,
-      signerBump: 0,
+      networkBump: 0,
```

Comment:
Generally if the previous comment about removing the bump is true, do a sweep of the code to remove it everywhere

Reply:
How should I set it then in the program side of things? Just have it default to a particular value?

Reply:
Oh I understand, never mind!